src/main/java/com/unascribed/fabrication/client/FabricationConfigScreen.java

```diff
@@ -1058,7 +1058,7 @@ private void color(int packed) {
 	}
 	
 	private void color(int packed, float alpha) {
-		GlStateManager.color4f(((packed>>16)&0xFF)/255f, ((packed>>8)&0xFF)/255f, ((packed>>0)&0xFF)/255f, alpha);
+		GlStateManager.color4f(((packed>>16)&0xFF)/255f, ((packed>>8)&0xFF)/255f, (packed&0xFF)/255f, alpha);
```

Comment:
Again, the zero-shift is intentional. The compiler optimizes this out immediately, there is no reason to remove it.